cs-amazon/src/main/java/io/cloudslang/content/amazon/actions/cloudformation/CreateStackAction.java

```diff
@@ -1,115 +1,131 @@
+import static org.apache.commons.lang3.StringUtils.defaultIfEmpty;
+
+public class CreateStackAction {
+    /**
+     * Creates AWS Cloud Formation Stack in sync mode using AWS Java SDK
+     *
+     * @param identity          Access key associated with your Amazon AWS or IAM account.
+     *                          Example: "wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY"
+     * @param credential        Secret access key ID associated with your Amazon AWS or IAM account.
+     * @param proxyHost         Optional - proxy server used to connect to Amazon API. If empty no proxy will be used.
+     *                          Default: ""
+     * @param proxyPort         Optional - proxy server port. You must either specify values for both proxyHost and
+     *                          proxyPort inputs or leave them both empty.
+     *                          Default: ""
+     * @param proxyUsername     Optional - proxy server user name.
+     *                          Default: ""
+     * @param proxyPassword     Optional - proxy server password associated with the proxyUsername input value.
+     *                          Default: ""
+     * @param region            AWS region name
+     *                          Example: "eu-central-1"
+     * @param stackName         Stack name to create
+     *                          Example: "MyStack"
+     * @param templateBody      AWS Cloud Formation template body in JSON or YAML format
+     * @param parameters        AWS Cloud Formation template parameters in key value format, one key=value per line
+     * @return                  A map with strings as keys and strings as values that contains: outcome of the action, returnCode of the
+     *                          operation, or failure message and the exception if there is one
+     */
+    @Action(name = "Create AWS Cloud Formation Stack",
+            outputs = {
+                    @Output(Outputs.RETURN_CODE),
+                    @Output(Outputs.RETURN_RESULT),
+                    @Output(Outputs.EXCEPTION)
+            },
+            responses = {
+                    @Response(text = Outputs.SUCCESS, field = Outputs.RETURN_CODE, value = Outputs.SUCCESS_RETURN_CODE,
+                            matchType = MatchType.COMPARE_EQUAL, responseType = ResponseType.RESOLVED),
+                    @Response(text = Outputs.FAILURE, field = Outputs.RETURN_CODE, value = Outputs.FAILURE_RETURN_CODE,
+                            matchType = MatchType.COMPARE_EQUAL, responseType = ResponseType.ERROR)
+            }
+    )
+    public Map<String, String> execute(
+            @Param(value = IDENTITY,   required = true)  String identity,
+            @Param(value = CREDENTIAL, required = true, encrypted = true)  String credential,
+            @Param(value = REGION,     required = true)  String region,
+            @Param(value = PROXY_HOST)                   String proxyHost,

```

Comment:
There's no need for more than one space between @Param and type. Also, the parameters can be **final**.